science/ncview/Portfile

```diff
@@ -4,8 +4,7 @@ PortSystem              1.0
 PortGroup               conflicts_build 1.0
 
 name                    ncview
-version                 2.1.7
-revision                2
+version                 2.1.8
 maintainers             gmail.com:thiagoveloso openmaintainer
```

Comment:
Oh, and we probably also need you to change this to 

`maintainers             {@thiagoveloso gmail.com:thiagoveloso} openmaintainer`